app/controllers/admin/participants_controller.rb

```diff
@@ -8,6 +8,7 @@ class ParticipantsController < Admin::BaseController
     before_action :load_participant, except: :index
     before_action :historical_induction_records, only: :show, unless: -> { @participant_profile.npq? }
     before_action :latest_induction_record, only: :show, unless: -> { @participant_profile.npq? }
+    before_action :event_list, only: :show
 
     def show; end
```

Comment:
If this is just being set for `:show` it probably makes more sense to assign inside the `show` method.

Reply:
Seem to have linter issues when I try to do this